components/feature/addons/src/main/java/mozilla/components/feature/addons/amo/AddonCollectionProvider.kt

```diff
@@ -101,13 +110,13 @@ class AddonCollectionProvider(
         }
 
         return try {
-            fetchAvailableAddons(readTimeoutInSeconds)
+            fetchAvailableAddons(readTimeoutInSeconds, language)
         } catch (e: IOException) {
             logger.error("Failed to fetch available add-ons", e)
             if (allowCache) {
-                val cacheLastUpdated = getCacheLastUpdated(context)
+                val cacheLastUpdated = getCacheLastUpdated(context, language, useFallbackFile = true)
```

Comment:
@csadilek in this case, when we try to fetch add-ons and we have a network error, we always want to fallback to any localized json file if necessary, for this reason we use `useFallbackFile = true`.